modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -482,6 +482,13 @@ When adjusting row height through the row dialog, any height styles previously a
 [[removed]]
 == Removed
 
+=== Removed `force_hex_color` option, with the default now being all colors are forced to HEX format and as lower case.
```

Comment:
```suggestion
=== Removed `force_hex_color` option, with the default now being almost all colors are forced to HEX format and as lower case. With the exception being RGBA, they are left as is.
```

Reply:
That could be mentioned in the body rather than the heading I think

Reply:
Do we want to update the body then, and modify the title? Im happy to let you guys decide on this one.

Reply:
Leaving the title as is, as the `With the exception being RGBA, they are left as is` was added to the body.